### ISSUE TITLE
Reg: E-Learn App

### ISSUE THREAD

**Issue by selvagee** [MangeCat-Studio/E-learn#14]
Hi,

I can't find routes folder in your E-Learn App

**Comment by AndrewKralovec** [comment]
Yeah, we are starting over from scratch on a different repo. Many of the features from the old repo has not been added yet. However, there a still quite a bit uploaded already. We are predicting the project to be done before this May. 
Here is a link to the new one 
https://github.com/SIU-CS-435/Alpha-Learning

**Comment by AndrewKralovec** [comment]
@Dreamwalker420  Should we remove this repo completely, or are we going to add the old E-learn back up here, at one point?
